### ISSUE TITLE
New entry - Mario's Picross 2 (USA)

### ISSUE THREAD

**Issue by ghost** [libretro-thumbnails/Nintendo_-_Game_Boy#56]
![Mario's Picross 2 (USA)](https://user-images.githubusercontent.com/105944840/169872033-142c5fd4-99d3-4ef8-aa40-e3ef5ac52353.png)

**Comment by xunkar** [comment]
This game was never released outside of Japan, there shouldn't be a box art for it. I'm interested in reading about everyone's opinion on the subject though.

By the way @masanasake you might be interested in joining up on this thread I made as I'm currently working through the GameBoy library to update most of the box arts. https://forums.libretro.com/t/thumbnails-repository-facelift/37458

**Comment by masanasake** [comment]
I had seen where someone's started cleaning up/taming the Gameboy games per your article. I added what I could, and have been cleaning up some of these manually to the best of my ability (Like opening them in paint and trying to remove stray marks/debris). I'm all for making Retroarch as good as it can be.

**Comment by masanasake** [comment]
Yeah, I get what you're saying. There are a few that I had scanned that had legit names that are recognized, and I used those where applicable. I tried to focus on ones that were officially recognized since they seemed like a given (like Super Mario Land 2 DX [Hack by toruzz]), but there are others, like Mother 3, where they've basically been accepted as the closest to an official release that we'll ever get. I tried not to get too obscure. No hate for this (I've never tried it), but I'd probably have drawn the line before this one: 
<img width="562" alt="image" src="https://user-images.githubusercontent.com/105944840/170526479-25de143d-6ba9-4688-87bb-0d52afd160a8.png">
